### DISCUSSION TITLE
Clicking Exit does nothing & Prevents Windows Shutdown

### DISCUSSION THREAD

**Discussion by TKithin** [Rem0o/FanControl.Releases#886] [Q&A]
Been having this issue a lot lately where the program is running and seems to be working fine but when I go to shutdown my computer it hangs and says Fan Control is Preventing Shut Down. And when I tell it cancel and try to Exit the program it just sits there and does nothing... I have managed to Exit the Program a few moments after start up and at other times so I am not entirely sure what the cause is.

![System Info1](https://user-images.githubusercontent.com/102504667/174732074-81563917-548d-42b4-8b59-3d50d8447df3.PNG)
![System Info2](https://user-images.githubusercontent.com/102504667/174732081-1717a093-2d85-4720-b25d-b01c5fdb21bf.PNG)

**Comment by Rem0o** [comment]
The one thing fancontrol prevents shutdown for is a prompt to save your config. Have you looked at the UI when that happens?

**Reply by TKithin** [reply]
This very well could be the cause, and after I've canceled shutdown and try and close the app it doesn't want to pop up the save config screen so I'm sitting there right clicking the task icon and choosing exit and its just sitting there... I have noticed it askes me kind of often (once a week maybe twice depending?) to save the config like something has changed and I'm not entirely sure what... could just be losing the sensors from HWINFO and then me reenabling them that is causing that... not sure... *shrugs* but i'll be more mindful of checking for it asking about saving before shutting down to see if that elevates this issue

**Comment by OceanOv** [comment]
Same problem
